openid-provider-commands-1_0.md

```diff
@@ -609,15 +618,15 @@ command_token=eyJhbGci ... .eyJpc3Mi ... .T3BlbklE ...
 
 ## Streaming Response
 
-The RP sends a Streaming Response to a Streaming Request. In a Streaming Response, the RP uses SSE to stream the Command Response as a sequence of events. If the RP receives a valid Command, it MUST sent the `HTTP/1.1 200 OK` response, followed by the following headers:
+The RP sends a Streaming Response to a Streaming Request. In a Streaming Response, the RP uses SSE to stream the Command Response as a sequence of events. If the RP receives a valid Command, it MUST sent the `200 OK` response, followed by the following headers:
```

Comment:
I suggest return back `HTTP` so it becomes `HTTP 200 OK` response (this was it doesn't matter if it's HTTP 1.1, 2 or 3).

```suggestion
The RP sends a Streaming Response to a Streaming Request. In a Streaming Response, the RP uses SSE to stream the Command Response as a sequence of events. If the RP receives a valid Command, it MUST sent the `HTTP 200 OK` response, followed by the following headers:
```

Reply:
As a side comment, HTTP 204 has [a special meaning in SSE](https://html.spec.whatwg.org/multipage/server-sent-events.html#:~:text=HTTP%20204%20No%20Content%20response%20code.) which makes a big difference with

> However, note that some Web frameworks will substitute an HTTP 204 No Content response for an HTTP 200 OK when the HTTP body is empty. Therefore, OPs should be prepared to also process an HTTP 204 No Content response as a successful response.

mentioned in the Command Response section.